bundle/pom.xml

```diff
@@ -730,6 +731,9 @@
             <version>6.0.0</version>
             <scope>provided</scope>
         </dependency>
-        
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
```

Comment:
Shouldn't this rather have scope "provided" because in fact the necessary packages are provided by AEM? Also you would then be able to remove the additional include below `bannedDependencies`.

Reply:
@kwin good catch! I'm kicking myself for not catching that!